dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenAPI-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpenAPI-1.0.properties

```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature provides support for the MicroProfile OpenAPI for enterprise Java.
+description=This feature provides support for the Eclipse MicroProfile OpenAPI for enterprise Java.
```

Comment:
Did we want to add "specification" after OpenAPI to be consistent with the other updates?